nms/app/packages/magmalte/app/components/lte/AddEditEnodebDialog.js

```diff
@@ -83,6 +83,9 @@ export default function AddEditEnodebDialog(props: Props) {
   );
   const [serialId, setSerialId] = useState<string>(editingEnodeb?.serial || '');
   const [tac, setTac] = useState(String(editingEnodeb?.config.tac || ''));
+  const [mme_ip, setMmeIP] = useState(
+    String(editingEnodeb?.config.mme_ip || '10.0.2.1'),
```

Comment:
Defaulting to @andreilee for this, but it seems iffy to me to define a hard coded string here, do we have a constants file for this to be defined?

Reply:
ok , i will make changes and git push.